src/demos/calcite-switch.html

```diff
@@ -21,6 +21,25 @@
   <body>
     <calcite-button href="/">Home</calcite-button>
     <h1>Calcite Switch</h1>
+
+    <calcite-label layout="inline">
+      Small
+      <calcite-switch scale="s"></calcite-switch>
+    </calcite-label>
+    <br>
+
+    <calcite-label layout="inline">
+      Medium
+      <calcite-switch scale="m"></calcite-switch>
+    </calcite-label>
+    <br>
+
+    <calcite-label layout="inline">
+      Large
+      <calcite-switch scale="l"></calcite-switch>
+    </calcite-label>
+    <br>
+
```

Comment:
Thanks for adding these!  I noticed they were missing on a recent bug I just fixed.

Reply:
sweet!